### PR TITLE
Remove docs around using HTTP-01 validation for LetsEncrypt

### PR DESCRIPTION
**Pull request by bengerman13**
HTTP-01 is currently broken, and not tested, so remove docs about using it

### CODE REVIEW

**Comment by siennathesane** [comment]
Let's reference that it's currently in a broken state and not actively tested instead of taking it down completely because we need to ensure we have incentive to fix it.

**Comment by bengerman13** [comment]
@mxplusb sounds good. Updated with your recommendation.

File: content/docs/services/cdn-route.md [hunk 1]
```diff
@@ -88,7 +88,10 @@ The maximum number of domains that can be associated with a single cdn-route ser
 
 ### How to set up DNS
 
-**Note:** If you are creating a new site on cloud.gov or you are migrating an existing site to cloud.gov that can tolerate a small amount of downtime during the migration, you can skip the first step and proceed directly to [Create CNAME record(s)](#step-2-create-cname-record-s)
+**Note:** Due to changes in how CloudFront processes requests, we currently only offer certificate provisioning via DNS challenges. This means that users can no longer skip step 1 below.
+We are investigating ways to bring this feature back.
```
Comment (siennathesane) [inline content/docs/services/cdn-route.md]: Can you add something like this:

> If you have a use case where DNS configurations are not available for you, please reach out to us at support@cloud.gov.

It's a good opportunity for us to find out how many users really use the HTTP-01 challenges.

**Review by siennathesane — CHANGES_REQUESTED** [review]
Sorry for being nitpicky.

File: content/docs/services/cdn-route.md (hunk 1 above)
Comment (bengerman13) [reply content/docs/services/cdn-route.md]: updated.

**Review by siennathesane — APPROVED** [review]
LGTM

**Pull request merged**